include/swift/SIL/SILModule.h

```diff
@@ -132,6 +132,10 @@ class SILModule {
   /// The list of SILVTables in the module.
   VTableListType vtables;
 
+  /// This is a cache of vtable entries for quick look-up
+  llvm::DenseMap<const SILVTable *, llvm::DenseMap<SILDeclRef, SILFunction *>>
+      VTableEntryCache;
+
```

Comment:
Actually ... I'm not sure about this. Probably it would be more efficient, but I don't know if this is significant.